"rectype","issueid","project_owner","project_name","actor","time","text","action","title"
"issue_title","260","pycqa","astroid","pylint-bot","2015-11-22 04:53:18","Originally reported by: **BitBucket: [ceridwenv](http://bitbucket.org/ceridwenv), GitHub: @ceridwen**

---

Previously, the code for building mock ASTs for enums was directly inserting Instance objects into the locals dictionary for enums.  I changed this to wrap them in InterpreterObjects, but this is now breaking the tests.  I'm not quite sure what the best way to handle this situation is.

---
- Bitbucket: https://bitbucket.org/logilab/astroid/issue/260
","start issue","[2.0 bugs] Enum mock ASTs"
"issue_closed","260","pycqa","astroid","pylint-bot","2015-12-09 05:55:57","","closed issue","[2.0 bugs] Enum mock ASTs"
"issue_comment","260","pycqa","astroid","pylint-bot","2015-11-23 08:31:16","_Original comment by_ **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**:

---

Taking a look at the test test_int_enum for instance, it seems that it doesn't use inference after obtaining the attribute with .getattr. In that case, just change it to use either igetattr or to infer what's retrieving, this should fix the problem. Wrapping these in InterpreterObjects makes sense.
","",""
"issue_comment","260","pycqa","astroid","pylint-bot","2015-11-23 16:28:53","_Original comment by_ **BitBucket: [ceridwenv](http://bitbucket.org/ceridwenv), GitHub: @ceridwen**:

---

Enum tests changed to use inference properly in 054966c1572d
","",""
